types/vote_set_test.go

```diff
@@ -519,6 +523,112 @@ func TestVoteSet_MakeCommit(t *testing.T) {
 	}
 }
 
+func TestVoteSet_LLMQType_50_60(t *testing.T) {
+	testCases := []struct {
+		llmqType      btcjson.LLMQType
+		numValidators int
+		threshold     int
+	}{
+		{
+			llmqType:      btcjson.LLMQType(0), // "tendermint" algorithm
+			numValidators: 40,
+			threshold:     int(math.Floor(2.0/3.0*40)) + 1,
+		},
+		{
+			llmqType:      btcjson.LLMQType_50_60,
+			numValidators: 35,
+			threshold:     30,
+		},
+		{
+			llmqType:      btcjson.LLMQType(0),
+			numValidators: 50,
+			threshold:     34,
+		},
+		{
+			llmqType:      btcjson.LLMQType_50_60,
+			numValidators: 50,
+			threshold:     30,
+		},
+	}
+
+	for ti, tt := range testCases {
+		name := strconv.Itoa(ti)
+		t.Run(name, func(t *testing.T) {
+			height, round := int64(1), int32(0)
```

Comment:
what if to declare `height` and `round` out of this function? or to move its inside `randVoteSetWithLLMQType`?